app/javascript/components/admin/AdminTable.js

```diff
@@ -465,6 +500,20 @@ class AdminTable extends React.Component {
     );
   };
 
+  // /**
```

Comment:
is there a reason this is still in here?

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/805/commits/b02f3a00163c7a98b7c07659635b1495dcdc5c78